### PR TITLE
Fix/materials filters

### PR DESCRIPTION
**Pull request by PavloJedi**

### CODE REVIEW

File: front-end/src/hooks/useFetchUnits.js [hunk 1]
```diff
@@ -12,14 +13,20 @@ const useFetchUnits = (isEdit, searchUnit, selectedLevel) => {
       try {
         if (!isEdit) {
           setLoading(true);
-          const units = await getUnitsByLevel(selectedLevel);
+          const units =
```
Comment (andriydzhrvts) [inline front-end/src/hooks/useFetchUnits.js:19]: Why you use `getUnitsByLevel` from question service in materials?

File: front-end/src/hooks/useFetchUnits.js [hunk 2]
```diff
@@ -12,14 +13,20 @@ const useFetchUnits = (isEdit, searchUnit, selectedLevel) => {
...
           setData(units);
           setLoading(false);
           setError(null);
         }
         if (isEdit && searchUnit !== '') {
           setLoading(true);
-          const questionsFromInput = await getQuestionsByUnitName({ searchUnit });
+          const questionsFromInput =
+            materialsOrQuestions === 'question'
+              ? await getQuestionsByUnitName({ searchUnit })
```
Comment (andriydzhrvts) [inline front-end/src/hooks/useFetchUnits.js:31]: The same as ^

File: front-end/src/pages/Questions/Questions.jsx [hunk 3]
```diff
@@ -58,7 +58,9 @@ const Questions = () => {
 
   const searching = searchUnitName === '' ? searchUnitName : searchUnit;
 
-  const { data, loading } = useFetchUnits(isEdit, searching, selectedLevel);
+  const { data, loading } = useFetchUnits(isEdit, searching, selectedLevel, 'question');
+
+  console.log(data);
```
Comment (andriydzhrvts) [inline front-end/src/pages/Questions/Questions.jsx]: Remove log

File: front-end/src/hooks/useFetchUnits.js (hunk 1 above)
Comment (PavloJedi) [reply front-end/src/hooks/useFetchUnits.js:19]: I wrote a condition to get a request for the materials in my component, and i get request for materials only

File: front-end/src/pages/Questions/Questions.jsx (hunk 3 above)
Comment (PavloJedi) [reply front-end/src/pages/Questions/Questions.jsx]: removed

**Review by andriydzhrvts — APPROVED** [review]

**Pull request merged**